frame/support/src/traits/tokens/fungible.rs

```diff
@@ -32,41 +32,65 @@ pub use imbalance::{Imbalance, HandleImbalanceDrop, DebtOf, CreditOf};
 pub trait Inspect<AccountId> {
 	/// Scalar type for representing balance of an account.
 	type Balance: Balance;
+
 	/// The total amount of issuance in the system.
 	fn total_issuance() -> Self::Balance;
+
 	/// The minimum balance any single account may have.
 	fn minimum_balance() -> Self::Balance;
+
 	/// Get the balance of `who`.
 	fn balance(who: &AccountId) -> Self::Balance;
+
+	/// Get the maximum amount that `who` can withdraw/transfer successfully.
+	fn withdrawable_balance(who: &AccountId) -> Self::Balance { Self::balance(who) }
```

Comment:
```suggestion
	fn burnable_balance(who: &AccountId) -> Self::Balance { Self::balance(who) }
```

You changed the API names in mutate, so these names should be updated accordingly